src/_i18n/es.yml

```diff
@@ -141,9 +141,14 @@ publications-post:
 services:
   title: Software de calidad entregado efectivamente
   subtitle: Lideres reconocidos en continuous delivery y prácticas de calidad y pericia.
+  we-acd: We advise, we coach, we deliver
+  software-craftsmanship-pioneers: Somos pioneros en Software Craftsmanship. Somos conocidos por nuestra experiencia en prácticas XP, Continuous Delivery, Software Architecture, soluciones Cloud e Infraestructura. Aprovechamos nuestra experiencia para ofrecer un servicio a medida de nuestros clientes.
+  we-advise-subtitle: Te asesoramos en la entrega de Software de calidad de forma efectiva.
+  we-coach-subtitle: Te entrenamos en habilidades de diseño de Software y delivery.
```

Comment:
delivery -> desarrollo

Reply:
In this case I think is more entrega than desarrollo... but I dislike both, I think is better to leave it in english